.travis.yml

```diff
@@ -20,10 +18,14 @@ php:
 
 matrix:
   fast_finish: true
+  allow_failures:
+    - php: hhvm
```

Comment:
why ? it is passing in master, so there is no reason to allow it to fail. If your changes are breaking things, you should fix your contribution instead